MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -28,24 +28,24 @@
             <div class="card-body" style="background-color:white">
                 <h2 class="card-title">Mohini Chakravarty</h2>
                 <p class="card-text">
-                    Scrum Master
+                    <div> Scrum Master </div> 
                 </p>
             </div>
         </div>
         <div class="col-sm-5">
-            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Mohini">
+            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Image of Mohini, the scrum master of the team" />
         </div>
     </div>
     <hr />
     <div class="row no-gutters">
         <div class="col-sm-5">
-            <img src="/images/Gaytri.png" class="card-img-top h-75" alt="Gaytri">
+            <img src="/images/Gaytri.png" class="card-img-top h-75" alt="Image of Gayatri who is the backend developer of the team" />
         </div>
         <div class="col-sm-7">
             <div class="card-body" style="background-color:white">
                 <h2 class="card-title">Gaytri Khatwani</h2>
                 <p class="card-text">
-                    Back End Developer
+                    <div>Back End Developer</div>
```

Comment:
Maintaining uniformity in the code as uniformity is very important in code review.

Reply:
Ditto comments above.